software/localess.yml

```
@@ -0,0 +1,13 @@
+name: "Localess"
+website_url: "https://localess.org/home"
+source_code_url: "https://github.com/Lessify/localess"
+description: "Powerful translation management and content management system. Manage and translate your website or app content into multiple languages, and it uses Artificial Intelligence to translate faster."
```

The last sentence changes tense in the middle. Perhaps something like this would be better:
```suggestion
description: "Powerful translation management and content management system. Manage and translate your website or app content into multiple languages, using Artificial Intelligence to translate faster."
```
Also, shouldn't Artificial Intelligence be abbreviated to A.I. or lowercased to artificial intelligence?